modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/utils/GetTotalBudget.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {useMemo} from 'react';
+
+import MDFRequest from '../../../../../common/interfaces/mdfRequest';
+import MDFRequestActivity from '../../../../../common/interfaces/mdfRequestActivity';
+
+const GetTotalBudget = (values: MDFRequest) => {
```

Comment:
Hey @DiogoNo. We cannot use hook inside function. You need to rename it as a custom hook: `useTotalBudget`